Sources/CodableKit/Reflection/HiLoDecoder.swift

```diff
@@ -61,105 +62,107 @@ struct HiLoDecoder: Decoder {
             properties.append(property)
         }
     }
-    
+
     private struct KeyedDecoder<Key>: KeyedDecodingContainerProtocol where Key: CodingKey {
         let allKeys: [Key] = []
         let ctx: Context
         let codingPath: [CodingKey]
-        
+
         init(_ ctx: Context, codingPath: [CodingKey]) {
             self.ctx = ctx
             self.codingPath = codingPath
         }
-        
+
         func contains(_ key: Key) -> Bool {
             ctx.nextIsOptional = true
             return true
         }
-        
+
         func decodeNil(forKey key: Key) throws -> Bool {
             ctx.nextIsOptional = true
             return false
         }
-        
+
         func decode<T>(_ type: T.Type, forKey key: Key) throws -> T where T : Decodable {
             return try SingleValueDecoder(ctx, codingPath: codingPath + [key]).decode(T.self)
         }
-        
+
         func nestedContainer<NestedKey>(keyedBy type: NestedKey.Type, forKey key: Key) throws -> KeyedDecodingContainer<NestedKey> where NestedKey : CodingKey {
             return .init(KeyedDecoder<NestedKey>(ctx, codingPath: codingPath + [key]))
         }
-        
+
         func nestedUnkeyedContainer(forKey key: Key) throws -> UnkeyedDecodingContainer {
             return UnkeyedDecoder(ctx, codingPath: codingPath + [key])
         }
-        
+
         func superDecoder() throws -> Decoder {
             return HiLoDecoder(ctx, codingPath: codingPath)
         }
-        
+
         func superDecoder(forKey key: Key) throws -> Decoder {
             return HiLoDecoder(ctx, codingPath: codingPath + [key])
         }
     }
-    
+
     private struct UnkeyedDecoder: UnkeyedDecodingContainer {
         var count: Int?
         var isAtEnd: Bool
         var currentIndex: Int
         var key: CodingKey {
-            return StringCodingKey(currentIndex.description)
+            return StringCodingKey.init(intValue: currentIndex)!
```

Comment:
```suggestion
            return StringCodingKey(currentIndex)
```
no need for `intValue` plus force unwrap here.